services/order.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: Optional[str] = None
+) -> None:
+    with transaction.atomic():

```

Comment:
You should use `transaction.atomic` as decorator instead of context manager. It makes your code more readable